### PR TITLE
Update instructions for cluster_protector.

### PR DESCRIPTION
**Pull request by mrtyler**
Based on [this comment](https://github.com/gpii-ops/gpii-infra/pull/199#issuecomment-463017515), Sergey tried this out while deploying https://github.com/gpii-ops/gpii-infra/pull/285 and found it worked. I did some more testing in my dev cluster.

### CODE REVIEW

**Comment by natarajaya** [comment]
LGTM!

File: gcp/modules/gke-cluster/main.tf [hunk 1]
```diff
@@ -68,9 +68,18 @@ resource "random_id" "cluster_protector" {
     # scratch or to change a parameter like 'oauth_scopes' that requires
     # cluster re-creation):
     #
+    # * cd gcp/live/ENV
+    # * rake sh"[sh -c \"cd /project/live/dev/k8s/cluster && terragrunt state rm random_id.cluster_protector\"]"
```
Comment (natarajaya) [inline gcp/modules/gke-cluster/main.tf]: Should this be `cd /project/live/ENV/k8s/cluster`?

**Comment by amatas** [comment]
LGTM

**Comment by stepanstipl** [comment]
lgtm

**Pull request merged**